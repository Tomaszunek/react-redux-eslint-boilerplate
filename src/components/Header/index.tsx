import React, { FunctionComponent } from 'react';

const Header: FunctionComponent<{}> = () => (
  <header>
    <div >
      <h1 >React Typescript Starter</h1>      
    </div>
  </header>
);

export default Header;
