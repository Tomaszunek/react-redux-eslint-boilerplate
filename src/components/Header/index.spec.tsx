import React from 'react';
import { create } from 'react-test-renderer';
import { shallow } from 'enzyme';
import Header from './index';

describe('App component', () => {
    it('starts with a count of 0', () => {
        const tree = create(<Header />).toJSON();
        expect(tree).toMatchSnapshot();
    });
    it('decrements count by 1 when the decrement button is clicked', () => {
        const wrapper = shallow(<Header />);
        const decrementBtn = wrapper.find('span');
        expect(decrementBtn.length).toEqual(0);
      });
});