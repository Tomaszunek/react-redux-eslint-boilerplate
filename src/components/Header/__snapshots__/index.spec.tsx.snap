// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component starts with a count of 0 1`] = `
<header>
  <div>
    <h1>
      React Typescript Starter
    </h1>
  </div>
</header>
`;
